pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+target-version = ["py34", "py35", "py36", "py37", "py38"]
```

Comment:
Why is it needed to specify the versions explicitly?

Reply:
Because otherwise black may end up guessing wrong, and producing syntax that is an error with 3.4. Why black wants all versions instead of the lowest is kind of beyond me, but that's how it is :P